<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMain</class>
 <widget class="QMainWindow" name="OMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Garden</string>
  </property>
  <property name="locale">
   <locale language="Polish" country="Poland"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_file"/>
   <addaction name="actionOpen_file"/>
   <addaction name="actionClose_file"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_built_ins"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/Icons/document-open.png</normaloff>:/icons/Icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_file">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/Icons/document-new.png</normaloff>:/icons/Icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit_built_ins">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/Icons/insert-table.png</normaloff>:/icons/Icons/insert-table.png</iconset>
   </property>
   <property name="text">
    <string>Edit built-ins</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionClose_file">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/Icons/document-close.png</normaloff>:/icons/Icons/document-close.png</iconset>
   </property>
   <property name="text">
    <string>Close file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/Icons/application-exit.png</normaloff>:/icons/Icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
